terraform/amazon/modules/vault/inputs.tf

```diff
@@ -30,6 +30,8 @@ variable "vault_root_size" {}
 
 variable "vault_data_size" {}
 
+variable "vault_volume_encryped" {}
```

Comment:
Use the encrypted directly in the templates, as almost all nodes/instances should be templated in our terraform code